lib/core/mount.js

```diff
@@ -121,6 +121,8 @@ function mount(mountPath, parentApp, events) {
 	var sessionStorePromise;
 
 	if (sessionStore) {
+		sessionOptions.store = sessionStore(session);
+	} else if (sessionStore) {
```

Comment:
This is actually mean to be
```
+	if (typeof sessionStore === 'function') {
+		sessionOptions.store = sessionStore(session);
+	} else if (sessionStore) {
```

... but it's fine, I'll merge + fix it myself, want to get this in.